LayoutTests/fast/css/resize-textarea-align-content.html

```diff
@@ -24,9 +25,9 @@
 <script>
 function drag(startX, startY, destX, destY) {
     const actions = new test_driver.Actions()
-        .pointerMove(startX - 6, startY - 7)
+        .pointerMove(startX - 22, startY - 23)
         .pointerDown()
-        .pointerMove(destX - 6, destY - 7)
+        .pointerMove(destX - 22, destY - 23)
```

Comment:
Would be good to add a comment about how these values were calculated

Reply:
Done.